packages/@tinacms/react-sidebar/src/sidebar.ts

```diff
@@ -17,13 +17,14 @@ limitations under the License.
 */
 
 import { EventBus, Callback } from '@tinacms/core'
+import { NoFormsPlaceholder } from './components/NoFormsPlaceHolder'
 import * as React from 'react'
 
 export interface SidebarStateOptions {
   hidden?: boolean
   position?: SidebarPosition
   buttons?: SidebarButtons
-  emptyState?: React.ElementType
+  placeholder?: React.ReactNode
```

Comment:
I think I would rather pass the component than a node. Something like:

```suggestion
  placeholder?: React.FC
```

Reply:
Besides that this great! 🙌

Reply:
Yep sounds good!